dashboards-observability/public/components/event_analytics/explorer/save_panel/save_panel.tsx

```diff
@@ -52,12 +53,14 @@ export const SavePanel = ({
   curVisId,
   spanValue,
   setSubType,
+  metricMeasure,
   setMetricMeasure,
   setMetricLabel,
+  metricChecked,
 }: ISavedPanelProps) => {
   const [options, setOptions] = useState([]);
   const [checked, setChecked] = useState(false);
```

Comment:
I think this variable is not required. You can just use `metricChecked`

Reply:
checked state value is to handle the switch locally, rather than pulling in the setMetricChecked all the way from explorer.